examples/log-errors/scripts/log-errors.js

```diff
@@ -0,0 +1,4 @@
+window.unhandledErrors = [];
```

Comment:
Unhanded errors are errors that are not handled by the client code. You have an error handler - `window.onerror`. So, these errors are handled.
`window.capturedErrors` is a better name.

Don't place the script in the separate folder because it is only one.
Rename to `examples/extended-error-tracking/setup-collecting-errors.js`